openvalidators/prompts.py

```diff
@@ -119,6 +119,16 @@ def find_unique_tags(input_text: str):
 # Request a follow-up question given a preceding context.
 followup_request_template = "Ask one relevant and insightful question about the preceding context."
 
+# ==============================
+# === Augmentation request prompt ===
+# ==============================
+
+# Request an augmentation given a preceding context.
+
+augment_request_template = "Summarize the preceding context at"
+school_levels = ['elementary school','middle school', 'highschool', 'university', 'graduate school']
```

Comment:
As per the conversation with @Eugene-hu , I think these school_levels are too basic and too close together (will a response truly be that different between elementary and middle school?) 

We should add more randomization here. "Explain why this summarization is correct or incorrect", "Summarize the preceding text in pig latin", "Summarize the preceding context at a level that my grandmother can understand" etc.